packages/material-ui/src/styles/createPalette.test.js

```diff
@@ -99,12 +99,6 @@ describe('createPalette()', () => {
   describe('augmentColor', () => {
     const palette = createPalette({});
 
-    it('should throw when the input is invalid', () => {
```

Comment:
Could you adjust this so that we know what error is logged for which input?

Reply:
Basically you want a new test case, ok.

Reply:
While we're at it: How does removing it relate to the SO question? That one didn't mention the custom error.

Reply:
I have removed the exception to catch the problem higher in the method call chain. This removed test will be replaced with a new one, and a new test will be added for the new handled case.